src/com/oltpbenchmark/util/SQLUtil.java

```diff
@@ -352,7 +352,7 @@ public static String getInsertSQL(Table catalog_tbl) {
      * @return
      */
     public static String getInsertSQL(Table catalog_tbl, boolean escape_names, int...exclude_columns) {
-        return getInsertSQL(catalog_tbl, false, false, 1, exclude_columns);
+        return getInsertSQL(catalog_tbl, true, false, 1, exclude_columns);
```

Comment:
This is Clickhouse specific, right? What part of the code needed this?

Reply:
Yes this is so that the column names appear in the insert statements since we added two columns per table (we use the default values for those columns).